src/resources/Microsoft.Authorization/policyDefinitions/Audit-AzureHybridBenefit.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "Audit-AzureHybridBenefit",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "apiVersion": "2021-06-01",
+  "scope": null,
+  "properties": {
+    "policyType": "Custom",
+    "displayName": "Enable AHUB for eligible VMs",
```

Comment:
`Enable` isn't accurate for the displayName. `Audit` would fit better to what the policy does.